integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -617,8 +663,7 @@ def test_hash_multiple_mode_query_avg_distincts(data_gen, conf):
 @incompat
 @pytest.mark.parametrize('data_gen', _init_list_no_nans, ids=idfn)
 @pytest.mark.parametrize('conf', get_params(_confs, params_markers_for_confs), ids=idfn)
-@pytest.mark.parametrize('parameterless', ['true', pytest.param('false', marks=pytest.mark.xfail(
-    condition=not is_before_spark_311(), reason="parameterless count not supported by default in Spark 3.1+"))])
```

Comment:
why was this removed?

Reply:
I am sorry. This option shouldn't be removed. I added it back.